# Aurora sensor data
geometry_msgs/Point position
geometry_msgs/Quaternion orientation
float64 error
bool visible
uint32 port_handle